scripts/configure.py

```diff
@@ -751,7 +753,7 @@ def failed():
   parser.add_argument('--web','--webserver', action='store_true', default=False,
     help="Install and configure webserver")
   parser.add_argument('--restart-updater', action='store_true', default=False,
-    help="""Stop the updater if it is running and start the updated one. \
+    help="""Restart AmpliPis OS and the updater itself. \
```

Comment:
```suggestion
    help="""Restart AmpliPi's OS and the updater itself. \
```
What do you think of adding '--restart' to this?

Reply:
This gets tricky. The web updater and the configure.py could get out of sync.

Reply:
We could make restart and restart updater aliases?